docs/extend/modular-resources/create/_index.md

```diff
@@ -46,12 +46,13 @@ Provide this as a file inside of your module, <file>my_modular_resource.go</file
    - Find the relevant `viam-server` client interface as `<resource-name>/client.go` or `<resource-name>/client.py` on [Viam's GitHub](https://github.com/viamrobotics/rdk/blob/main/).
    - For example, the base client is defined in [<file>rdk/components/base/client.go</file>](https://github.com/viamrobotics/rdk/blob/main/components/base/client.go).
    - Base your edits to <file>my_modular_resource.go</file> or <file>my_modular_resource.py</file> on this first file.
+   - Name your model according to the namespace of the built-in API you are implementing. For more information see [Model namespace](/extend/modular-resources/key-concepts/#namespace-1).
```

Comment:
```suggestion
   - Name your model according to the namespace of the built-in API you are implementing.
   For more information see [Model namespace](/extend/modular-resources/key-concepts/#namespace-1).
```
Beep boop my name is Newline Linter like Sesame

Reply:
🤦 Thank you!!